test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerTests.cs

```diff
@@ -79,11 +79,30 @@ public void CreateTokenThrowsNullArgumentException()
             Assert.Throws<ArgumentNullException>(() => handler.CreateToken("Payload", Default.SymmetricEncryptingCredentials, (Dictionary<string, object>) null));
         }
 
-        [Fact]
-        public void ValidateTokenValidationResult()
+        [Theory, MemberData(nameof(TokenValidationClaimsTheoryData))]
+        public void ValidateTokenValidationResult(JsonWebTokenTheoryData theoryData)
         {
             TestUtilities.WriteHeader($"{this}.ValidateTokenValidationResult");
+            var tokenValidationResult = theoryData.TokenHandler.ValidateToken(theoryData.AccessToken, theoryData.ValidationParameters);
+            Assert.Equal(tokenValidationResult.Claims, TokenUtilities.CreateDictionaryFromClaims(tokenValidationResult.ClaimsIdentity.Claims));
+        }
+
+        [Theory, MemberData(nameof(TokenValidationClaimsTheoryData))]
+        public void ValidateTokenDerivedHandlerValidationResult(JsonWebTokenTheoryData theoryData)
+        {
+            TestUtilities.WriteHeader($"{this}.ValidateTokenValidationResult");
+            var derivedJsonWebTokenHandler = new DerivedJsonWebTokenHandler(); 
+            var tokenValidationResult = theoryData.TokenHandler.ValidateToken(theoryData.AccessToken, theoryData.ValidationParameters);
+            var tokenValidationDerivedResult = derivedJsonWebTokenHandler.ValidateToken(theoryData.AccessToken, theoryData.ValidationParameters);
+            Assert.Equal(tokenValidationResult.Claims, TokenUtilities.CreateDictionaryFromClaims(tokenValidationResult.ClaimsIdentity.Claims));
+            Assert.Equal(tokenValidationDerivedResult.Claims, TokenUtilities.CreateDictionaryFromClaims(tokenValidationDerivedResult.ClaimsIdentity.Claims));
+            Assert.Equal(tokenValidationDerivedResult.Claims, tokenValidationDerivedResult.Claims);
```

Comment:
>Assert.Equal(tokenValidationDerivedResult.Claims, tokenValidationDerivedResult.Claims); [](http://example.com/codeflow?start=12&length=87)

copy and paste error? Seems like this likely isn't checking what it's supposed to be.